packages/core/admin/admin/src/components/UnstableGuidedTour/Step.tsx

```diff
@@ -111,13 +115,19 @@ const createStepComponents = (tourName: ValidTourName): Step => ({
     </Box>
   ),
 
-  Actions: ({ showStepCount = true, showSkip = false, to, ...props }) => {
+  Actions: ({ showStepCount = true, showSkip = false, to, trackedEvent, ...props }) => {
```

Comment:
I don't really like this as a prop here and on the TASK_CONTENT constant. 

What if we made a new shared constant like 
```
GUIDED_TOUR_TRACKING = {
 contentTypeBuilder: { create: 'didCreateGuidedTourContentTypeSchema', start: 'didStartGuidedTourContentTypeBuilder'}
}
```
Then we can look up the tracking with the tourName
```
const trackingEvent = GUIDED_TOUR_TRACKING['contentTypeBuilder'].create
```
I think this way if we need to add or update a tracking event it will only require a change in one place vs two